src/emitters/emitter.sqns.ts

```diff
@@ -685,7 +707,7 @@ export class SqnsEmitter implements IEmitter {
       });
       throw new Error(`Failed to parse message`);
     }
-    const listeners = this.topicListeners.get(message.eventName);
+    const listeners = this.getTopicListeners(message.eventName);
```

Comment:
Not passing consumer options, so we will always be having a single key for the listeners?